### DISCUSSION TITLE
Different characters support?

### DISCUSSION THREAD

**Discussion by bartolek153** [abecodes/tabout.nvim#34] [Q&A] [answered]
Hey, I would like to know if there's some way to 'tabout' from other specific characters. I mean, the ' ; ' at the end of lines (in C language for example). I tried doing changes at the config file but didn't work.

**Comment by abecodes** [comment, marked as answer]
Hi @bartolek153 
in general it works with all kind of characters, as long as they are part of a treesitter node. The catch is, that we need a start character and a end character to limit the area we want to tabout from, which we wont have with ';' alone.

**Reply by bartolek153** [reply]
I see, so as you said, Treesitter doesn't tell that you're inside something with the ' ; ' character right? Is there any approach to make it recognize this as a node?

**Reply by abecodes** [reply]
I have smth in place for when nodes don't work, but the difference for you would be, that the starting char would be the beginning of the line.

What exactly would be your usecase, I am not sure I understand completely...you want to tab out behind the ';' of the current line?
